packages/shared/src/components/drawers/Drawer.tsx

```diff
@@ -67,19 +75,20 @@ function BaseDrawer({
   return (
     <div
       className={classNames(
-        'fixed inset-0 z-max bg-overlay-quaternary-onion transition-opacity duration-300 ease-in-out',
+        'fixed inset-0 z-max transition-opacity duration-300 ease-in-out',
```

Comment:
We can even make this overlay wrapper be used in `ConditionalWrapper`. Though it might break the styles and unnecessarily add more dev time here. We can settle with what we have now. Your call.